deck.py

```diff
@@ -0,0 +1,60 @@
+""""Sets up the Deck class for a network Poker game"""
+
+
+from card import Card
+from random import shuffle
+
+
+class Deck:
+    def __init__(self, deck_of_cards):
+        self.deck_of_cards = deck_of_cards
```

Comment:
When someone uses this class it will look like:

```py
my_deck = Deck()
my_deck.deck_of_cards...
```

Every access will repeat the word `deck`. I would just call this `cards` it's obviously part of a deck already.